crates/ruma-events/src/call/member.rs

```diff
@@ -0,0 +1,501 @@
+                    "expires": 3600000,
+                    "foci_active": [
+                        {
+                            "livekit_alias": "1",
+                            "livekit_service_url": "https://livekit.com",
+                            "type": "livekit"
+                        }
+                    ],
+                    "membershipID": "0"
+                }
+            ]
+        });
+
+        assert_eq!(
+            call_member_event,
+            &serde_json::to_value(create_call_member_event_content()).unwrap()
+        );
+    }
+
+    #[test]
+    fn deserialize_call_member_event_content() {
+        let call_member_ev: CallMemberEventContent = CallMemberEventContent::new(vec![
+            Membership {
+                application: Application::Call(CallApplicationContent {
+                    call_id: "123456".to_owned(),
+                    scope: CallScope::Room,
+                }),
+                device_id: "THIS_DEVICE".to_owned(),
+                expires: Duration::from_secs(3600),
+                foci_active: vec![Focus::Livekit(LivekitFocus {
+                    alias: "room1".to_owned(),
+                    service_url: "https://livekit1.com".to_owned(),
+                })],
+                membership_id: "0".to_owned(),
+                created_ts: None,
+            },
+            Membership {
+                application: Application::Call(CallApplicationContent {
+                    call_id: "".to_owned(),
+                    scope: CallScope::Room,
+                }),
+                device_id: "OTHER_DEVICE".to_owned(),
+                expires: Duration::from_secs(3600),
+                foci_active: vec![Focus::Livekit(LivekitFocus {
+                    alias: "room2".to_owned(),
+                    service_url: "https://livekit2.com".to_owned(),
+                })],
+                membership_id: "0".to_owned(),
+                created_ts: None,
+            },
+        ]);
+
+        let call_member_ev_json = json!({
+            "memberships": [
+                {
+                    "application": "m.call",
+                    "call_id": "123456",
+                    "scope": "m.room",
+                    "device_id": "THIS_DEVICE",
+                    #[allow(clippy::unreadable-literal)]
+                    "expires": 3600000,
+                    "foci_active": [
+                        {
+                            "livekit_alias": "room1",
+                            "livekit_service_url": "https://livekit1.com",
+                            "type": "livekit"
+                        }
+                    ],
+                    "membershipID": "0",
+                },
+                {
+                    "application": "m.call",
+                    "call_id": "",
+                    "scope": "m.room",
+                    "device_id": "OTHER_DEVICE",
+                    #[allow(clippy::unreadable-literal)]
+                    "expires": 3600000,
```

Comment:
```suggestion
                    "expires": 3_600_000,
```

Reply:
why should we fix it like that?
I wanted it to still be 3600000 to make sure the test does exactly what we will get from the hs?

Reply:
It doesn't change anything to the actual value, it's just for readability.

Reply:
The `serde_json::Value` that comes out is the exact same, you can just put in underscores in numeral literals in Rust for readability, they don't affect the compiler output.